doc/sphinx/methods.rst

```diff
@@ -367,7 +367,7 @@ To calculate Bader charges from a cube file, a ``Volume`` object should be prepa
 DDEC6
 ----------------
 
-DDEC6 charges are stockholder-like charge partitioning method introduced in `[Manz2016] <https://doi.org/10.1039/C6RA04656H>`_. Proatom densities should be provided for DDEC6 method. Proatom densities generated using `horton`_ can be passed on as an argument for the constructor of the DDEC6 object. The DDEC6 algorithm requires many numerical integrations so a fine grid is necessary to obtain accurate results. Calculating the electron density on a fine grid using a Volume object is slow, therefore we recommend that electron densities are imported from cube files.
+DDEC6 charges are stockholder-like charge partitioning method introduced in `[Manz2016] <https://doi.org/10.1039/C6RA04656H>`_. Proatom densities should be provided for DDEC6 method. Proatom densities generated [#proatom]_ using `horton`_ can be passed on as an argument for the constructor of the DDEC6 object. The DDEC6 algorithm requires many numerical integrations so a fine grid is necessary to obtain accurate results. Calculating the electron density on a fine grid using a Volume object is slow, therefore we recommend that electron densities are imported from cube files.
```

Comment:
Does markdown care if each sentence is on a separate line? If not we should put a sentence per line. It would be easier for review. We can probably just do this in the future and check the other files in the documentation.

Reply:
FWIW I'm fine with multiple sentences per line - it's how we've written markdown everywhere. I think the issue is that GitHub can't diff. Is there another reason to break up paragraphs?